README.md

```diff
@@ -12,6 +12,26 @@ See [the demo file](demo/usage.html) for an usage example.
 Keep in mind that the captcha only works when used from a real domain and with a valid re-captcha key, so this file won't work if you just load it in your browser.
 
 
+Install
+-----
+
+### Bower
+
+```
+bower install --save angular-recaptcha
+```
+
+### npm
+
+```
+npm install --save angular-recaptcha
+```
+
+### Manual
+
+Download `release/angular-recaptcha.js` and/or `release/angular-recaptcha.min.js`.
```

Comment:
Shouldn't this be a link to the [latest release](https://github.com/VividCortex/angular-recaptcha/releases/latest) ?